deployments/upf-k8s.yaml

```diff
@@ -82,16 +82,16 @@ spec:
   - name: routes
     image: omecproject/upf-epc-bess:master-latest
     env:
-    - name: ENB_SUBNET
+    - name: ENB_GNB_SUBNET
```

Comment:
Are you expecting any helm charts changes? Could you please share that PR as well.

Reply:
No, there is no Helm Charts changes. These values are used when someone tries to deploy the UPF in a K8s cluster using the `upf-k8s.yaml` file.